x-pack/plugins/cases/server/authorization/authorization.test.ts

```diff
@@ -0,0 +1,977 @@
+            { id: '3', owner: 'other-owner' },
+          ],
+          operation: Operations.createCase,
+        })
+      ).resolves.not.toThrow();
+
+      expect(mockLogger.log.mock.calls).toMatchInlineSnapshot(`
+        Array [
+          Array [
+            Object {
+              "event": Object {
+                "action": "case_create",
+                "category": Array [
+                  "database",
+                ],
+                "outcome": "unknown",
+                "type": Array [
+                  "creation",
+                ],
+              },
+              "kibana": Object {
+                "saved_object": Object {
+                  "id": "100",
+                  "type": "cases",
+                },
+              },
+              "message": "User is creating cases [id=100] as owner \\"a\\"",
+            },
+          ],
+          Array [
+            Object {
+              "event": Object {
+                "action": "case_create",
+                "category": Array [
+                  "database",
+                ],
+                "outcome": "unknown",
+                "type": Array [
+                  "creation",
+                ],
+              },
+              "kibana": Object {
+                "saved_object": Object {
+                  "id": "3",
+                  "type": "cases",
+                },
+              },
+              "message": "User is creating cases [id=3] as owner \\"other-owner\\"",
+            },
+          ],
+        ]
+      `);
+    });
+  });
+
+  describe('getAuthorizationFilter', () => {
+    const feature = { id: '1', cases: ['a', 'b'] };
+
+    let securityStart: ReturnType<typeof securityMock.createStart>;
+    let featuresStart: jest.Mocked<FeaturesPluginStart>;
+    let auth: Authorization;
+
+    beforeEach(async () => {
+      securityStart = securityMock.createStart();
+      securityStart.authz.mode.useRbacForRequest.mockReturnValue(true);
+      securityStart.authz.checkPrivilegesDynamicallyWithRequest.mockReturnValue(
+        jest.fn(async () => ({
+          hasAllRequested: true,
+          username: 'super',
+          privileges: { kibana: [] },
+        }))
+      );
+
+      featuresStart = featuresPluginMock.createStart();
+      featuresStart.getKibanaFeatures.mockReturnValue(([feature] as unknown) as KibanaFeature[]);
+
+      auth = await Authorization.create({
+        request,
+        securityAuth: securityStart.authz,
+        getSpace: jest.fn(),
+        features: featuresStart,
+        auditLogger: new AuthorizationAuditLogger(mockLogger),
+        logger: loggingSystemMock.createLogger(),
+      });
+    });
+
+    it('throws and logs an error when there are no registered owners from plugins and security is disabled', async () => {
+      expect.assertions(2);
+
+      featuresStart.getKibanaFeatures.mockReturnValue([]);
+
+      auth = await Authorization.create({
+        request,
+        securityAuth: securityStart.authz,
```

Comment:
Thanks for catching that. I switch the test description to say `enabled` because we need security enabled otherwise it won't hit the if condition to make it throw an error.